server/historian/packages/historian-base/src/routes/git/blobs.ts

```diff
@@ -30,6 +30,7 @@ export function create(
 	revokedTokenChecker?: IRevokedTokenChecker,
 	denyList?: IDenyList,
 	ephemeralDocumentTTLSec?: number,
+	simplifiedCustomDataRetriever?: ISimplifiedCustomDataRetriever,
```

Comment:
I don't see this new simplifiedCustomDataRetriever is used anywhere within the function. Or I miss something?

Reply:
good catch, it should be passed in as parameter of createGitService(), I will update this.

Reply:
Only the initial summary creation flow needs this new property right? Then we probably don't need to add it everywhere?

Reply:
Your point is valid, only create() operation needs this info, updated accordingly.